packages/bruno-app/src/utils/exporters/openapi-spec.js

```
@@ -41,14 +90,48 @@ export const exportApiSpec = ({ variables, items, name }) => {
+    // Unknown URL — resolve fully and add operation-level server override
+    const resolvedUrl = interpolate(rawUrl, variables);
+    if (isValidUrl(resolvedUrl)) {
+      const urlDetails = new URL(resolvedUrl);
+      return { url: urlDetails.pathname, operationLevelServer: buildServerEntry(urlDetails.origin, variables) };
+    }
+
+    return { url: rawUrl, operationLevelServer: null };
+  };
+
   const generatePaths = () => {
     const _items = items.map((item) => {
```

Biome (2.3.14)

[error] This callback passed to map() iterable method should always return a value.

Change this return so that it returns a value.

Change this return so that it returns a value.

Change this return so that it returns a value.


(lint/suspicious/useIterableCallbackReturn)